include/sdf/ParserConfig.hh

```diff
@@ -174,6 +174,12 @@ class SDFORMAT_VISIBLE ParserConfig
   /// \brief Get the preserveFixedJoint flag value.
   public: bool URDFPreserveFixedJoint() const;
 
+  /// \brief set the enableNewWarnings flag to _enable value
+  public: void URDFSetNewWarnings(const bool _enable);
+
+  /// \brief get the enableNewWarnings flag value;
+  public: bool URDFEnableNewWarnings() const;
```

Comment:
```suggestion
  public: EnforcementPolicy URDFPolicy() const;
```

Reply:
Changed in [b05e7b7](https://github.com/gazebosim/sdformat/pull/1192/commits/b05e7b72b2b48277c026b7290166eaffc0cd074a)